iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/model/ServiceAccount.java

```diff
@@ -24,6 +24,7 @@
 import org.sonar.iac.common.checks.Trilean;
 
 public record ServiceAccount(
+  String path,
```

Comment:
Maybe call it `filePath`, to clearly indicate that it is related to the filesystem and not to some Kubernetes concept?